source/common/http/http1/codec_impl.cc

```diff
@@ -134,7 +134,7 @@ void ConnectionImpl::flushOutput() {
     reserved_current_ = nullptr;
   }
 
-  connection().write(output_buffer_);
+  connection().write(output_buffer_, false);
```

Comment:
optional: do you think it's worth a comment on the first write here and in H2 that we do not support half-close semantics for HTTP[2]?

Reply:
At this point, it is everything but tcpproxy, so we'd want to document http1, http2, redis, mongo, etc.  I say we just leave it on the documentation for Connection.